test/test_bulk_actions.py

```diff
@@ -237,3 +249,26 @@ def test_clear_incident_in_metadata(metadata):
     assert endpoints["fiberlink"][0]["events"][0]["is_up"] is True
     assert endpoints["fiberlink"][0]["events"][0]["clear_time"] == clear_time
     assert endpoints["fiberlink"][0]["events"][0]["properties"]["status"] == "Clear"
+
+
+@pytest.mark.django_db
+@patch("src.geant_argus.geant_argus.incidents.bulk_actions.clear_alarm")
+def test_bulk_clear_incidents(mock_clear_alarm, default_user):
+    # Mock data
+    actor = default_user
+    qs = [MagicMock(metadata={"status": "ACTIVE", "endpoints": {}})]
+    data = {"timestamp": timezone.now()}
+    clear_time = data["timestamp"].replace(tzinfo=None).isoformat()
+
+    # Mock clear_alarm to return True
+    mock_clear_alarm.return_value = True
+
+    # Call the function
+    incidents = bulk_clear_incidents(actor, qs, data)
+
+    # Assertions
+    assert len(incidents) == 1
+    assert incidents[0].metadata["status"] == "CLEAR"
+    assert incidents[0].metadata["clear_time"] == clear_time
+    mock_clear_alarm.assert_called_once_with(qs[0].source_incident_id, {"clear_time": clear_time})
+    qs[0].save.assert_called_once()
```

Comment:
I'm personally not a fan of the `.assert_*` methods. I'd rather just write `assert qs[0].save.call_count == 1`. The `assert_` methods are more idiomatic when using the unittest framework, while just `assert`ing is more idiomatic when using pytest. However it's a personal preference, so if you like those methods, go ahead :)